app/services/ExactTargetService.scala

```diff
@@ -172,6 +173,48 @@ class ExactTargetService(
       }
     }
   }
+
+  def enqueueRenewalEmail(
+    oldSub:Subscription[Plan.WeeklyPlan],
+    renewal: Renewal,
+    subscriptionDetails: String,
+    contact: Contact,
+    email: String,
+    customerAcceptance: LocalDate,
+    contractEffective: LocalDate): Future[Unit] = {
+
+    sealed trait Error {
+      def msg : String
+    }
+    case class ExceptionThrown(msg: String, e: Throwable) extends Error
+    case class SimpleError(msg :String) extends Error
+
+    def getPaymentMethod(accountId: AccountId): Future[\/[Error, PaymentMethod]] = paymentService.getPaymentMethod(oldSub.accountId).map(_.toRightDisjunction(SimpleError("No payment method found in account")))
+
+    def sendToQueue(row: GuardianWeeklyRenewalDataExtensionRow): Future[\/[Error, SendMessageResult]] = SqsClient.sendDataExtensionToQueue(Config.holidaySuspensionEmailQueue, row).map {
+      case Success(sendMsgResult) => \/-(sendMsgResult)
+      case Failure(e) => -\/(ExceptionThrown(s"Failed to enqueue ${oldSub.name.get}'s guardian weekly renewal email. Details were: " + row.toString, e))
+    }
+
+    val sentMessage = (for {
+      paymentMethod <- EitherT(getPaymentMethod(oldSub.accountId))
+      row = GuardianWeeklyRenewalDataExtensionRow(oldSub.name,
+        renewal.plan.name,
+        subscriptionDetails,
+        contact,
+        paymentMethod,
+        email,
+        customerAcceptance,
+        contractEffective)
+      sendMessage <- EitherT(sendToQueue(row))
+    } yield (sendMessage)).run
+
+   sentMessage.map{
+        case \/-(sendMsgResult) => logger.info(s"Successfully enqueued ${oldSub.name.get}'s guardian weekly renewal email.")
+        case -\/(SimpleError(msg)) => logger.error(msg)
+        case -\/(ExceptionThrown(msg, ex)) => logger.error(msg, ex)
```

Comment:
do we need to add any context to these error messages? i.e. `.withTrace(s"email for ${oldSub.name.get})`